content/docs/command-reference/plots/index.md

```diff
@@ -12,30 +12,18 @@ usage: dvc plots [-h] [-q | -v] {show,diff,modify} ...
 
 positional arguments:
   COMMAND
-    show         Generate plot from a metrics file.
-    diff         Plot differences in metrics between commits.
-    modify       Modify display properties of data-series plots (has no effect on image-type plots).
+    show        Generate plots from target files or plots definitions from `dvc.yaml` file.
+    diff        Show multiple versions of plot data by plotting them in a single image.
+    modify      Modify display properties of data-series stage plots (has no effect on image-type plots).
```

Comment:
Can we wrap these texts please? It's a good practice to always sanity check changes [on the review app](https://dvc-org-7086-flexible-p-d0hfxv.herokuapp.com/doc/command-reference/plots):

![image](https://user-images.githubusercontent.com/1477535/178393917-ad6ced17-8ca7-4538-8169-4bda0e8e2168.png)

Reply:
OK how about this
```suggestion
    show      Generate plots from target files or from `plots`
                definitions in `dvc.yaml`.
    diff      Show multiple versions of a plot by overlaying them
                in a single image.
    modify    Modify display properties of data-series plots
                defined in stages (has no effect on image plots).
```